src/symtseries.c

```diff
@@ -215,72 +215,49 @@ static const double dist_table[STS_MAX_CARDINALITY - 1][STS_MAX_CARDINALITY][STS
     }
 };
 
-static sts_symbol get_symbol(double value, unsigned int c) {
+static sts_symbol _get_symbol(double value, unsigned int c) {
```

Comment:
This way static methods are just more easily distinguishable for me (yeah, even though we have sts_* for non-static), can revert back easily though.